DocSum/tests/test_compose_on_rocm.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+IMAGE_REPO=${IMAGE_REPO:-"opea"}
+IMAGE_TAG=${IMAGE_TAG:-"latest"}
+echo "REGISTRY=IMAGE_REPO=${IMAGE_REPO}"
+echo "TAG=IMAGE_TAG=${IMAGE_TAG}"
+
+
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+export REGISTRY=${IMAGE_REPO}
+export TAG=${IMAGE_TAG}
+export DOCSUM_TGI_IMAGE="ghcr.io/huggingface/text-generation-inference:2.3.1-rocm"
+export DOCSUM_LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+export HOST_IP=${ip_address}
+export DOCSUM_TGI_SERVICE_PORT="8008"
+export DOCSUM_TGI_LLM_ENDPOINT="http://${HOST_IP}:8008"
+export DOCSUM_HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+export DOCSUM_LLM_SERVER_PORT="9000"
+export DOCSUM_BACKEND_SERVER_PORT="8888"
+export DOCSUM_FRONTEND_PORT="5552"
+export MEGA_SERVICE_HOST_IP=${ip_address}
+export LLM_SERVICE_HOST_IP=${ip_address}
+export BACKEND_SERVICE_ENDPOINT="http://${ip_address}:8888/v1/docsum"
+export DOCSUM_CARD_ID="card1"
+export DOCSUM_RENDER_ID="renderD136"
+
+function build_docker_images() {
+    cd "$WORKPATH"/docker_image_build
+    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+
+    echo "Build all the images with --no-cache, check docker_image_build.log for details..."
+    docker compose -f build.yaml build --no-cache > "${LOG_PATH}"/docker_image_build.log
```

Comment:
No need to build all the images, just build the images required by this test, https://github.com/opea-project/GenAIExamples/blob/82f418bf3439672ce570d05cbe2a1ca257b3d6b8/DocSum/tests/test_compose_on_gaudi.sh#L22

Reply:
done